tutorials/dlp-hybrid-inspect/src/main/java/com/example/dlp/HybridInspectSql.java

```diff
@@ -63,104 +64,215 @@
   private static final Logger LOG = Logger.getLogger(HybridInspectSql.class.getName());
   private static final Level LOG_LEVEL = Level.WARNING;
+  private static final String DISPLAY_VERSION = "0.2";
 
+  // Below are standard JDBC URLs for a select set of data sources. To add more sources, you will need to add the URLs here and update
+  // code methods below to use these. 
   public static final String JDBC_URL_POSTGRESQL = "jdbc:postgresql://%s/%s";
   public static final String JDBC_URL_MYSQL = "jdbc:mysql://%s/%s?useSSL=false&allowPublicKeyRetrieval=true";
-  public static final String JDBC_URL_CLOUDSQL = "jdbc:mysql://google/%s?cloudSqlInstance=%s&socketFactory=com.google.cloud.sql.mysql.SocketFactory&useSSL=false";
+  public static final String JDBC_URL_SQLSERVER = "jdbc:sqlserver://%s;database=%s;integratedSecurity=false;";
+
+  // The values below can be configured to shrink the size of an individual request. Please consider usage limits and quota when adjusting these
+  // https://cloud.google.com/dlp/limits
   private static final int MAX_REQUEST_BYTES = 480000; // max request size in bytes
   private static final int MAX_REQUEST_CELLS = 50000; // max cell count per request
 
   /**
    * Command line application to inspect data using the Data Loss Prevention Hybrid
    */
   public static void main(String[] args) throws Exception {
-    System.out.println("Cloud DLP HybridInspect: SQL via JDBC V0.2");
+    System.out.println("Cloud DLP HybridInspect: SQL via JDBC Script Version " + DISPLAY_VERSION);
+    // generate a UUID as a tracking number for the scan job. This is sent as a label in the Hybrid
+    // Inspect Request and used for tracking.
+    String runId = UUID.randomUUID().toString();
+
+    java.util.Date startDate = new java.util.Date();
+
+    System.out.println("-----------------------------------------------------------------");
+    System.out.println(" Start Run ID: " + runId);
+    System.out.println(" Start Time: " + startDate.toString());
+    System.out.println("-----------------------------------------------------------------");
+
     LOG.setLevel(LOG_LEVEL);
 
     Options opts = new Options();
-    Option sqlOption = createAndAddOptWithArg(opts, "sql", true);
-    Option databaseInstanceServer = createAndAddOptWithArg(opts, "databaseInstanceServer", true);
-    Option databaseInstanceServerDisplay = createAndAddOptWithArg(opts,
-        "databaseInstanceDescription", true);
-    Option databaseName = createAndAddOptWithArg(opts, "databaseName", true);
-    Option tableName = createAndAddOptWithArg(opts, "tableName", false);
-    Option databaseUser = createAndAddOptWithArg(opts, "databaseUser", true);
-    Option secretManagerResourceName = createAndAddOptWithArg(opts, "secretManagerResourceName",
+
+    // General options
+    Option hybridJobNameOption = createAndAddOptWithArg(opts, "hybridJobName", false);
+    Option hybridJobTriggerNameOption = createAndAddOptWithArg(opts, "hybridJobTriggerName", false);
```

Comment:
I think you can combine these options. They don't need a separate flag for each, just have the one flag support both.

Reply:
I'll join these